hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/dao/OutgoingEdgesDataFetcher.java

```diff
@@ -0,0 +1,54 @@
+package org.hypertrace.graphql.entity.dao;
+
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import javax.inject.Inject;
+import org.hypertrace.core.graphql.common.fetcher.InjectableDataFetcher;
+import org.hypertrace.core.graphql.deserialization.ArgumentDeserializer;
+import org.hypertrace.graphql.entity.schema.EdgeResultSet;
+import org.hypertrace.graphql.entity.schema.Entity;
+import org.hypertrace.graphql.entity.schema.EntityType;
+import org.hypertrace.graphql.entity.schema.argument.NeighborEntityScopeArgument;
+import org.hypertrace.graphql.entity.schema.argument.NeighborEntityTypeArgument;
+
+public class OutgoingEdgesDataFetcher extends InjectableDataFetcher<EdgeResultSet> {
+  public OutgoingEdgesDataFetcher() {
+    super(OutgoingEdgesDataFetcherImpl.class);
+  }
+
+  private static class OutgoingEdgesDataFetcherImpl
+      implements DataFetcher<CompletableFuture<EdgeResultSet>> {
+
+    private final ArgumentDeserializer argumentDeserializer;
+
+    @Inject
+    OutgoingEdgesDataFetcherImpl(ArgumentDeserializer argumentDeserializer) {
+      this.argumentDeserializer = argumentDeserializer;
+    }
+
+    @Override
+    public CompletableFuture<EdgeResultSet> get(DataFetchingEnvironment environment) {
+      return CompletableFuture.completedFuture(
+          environment
+              .<Entity>getSource()
+              .outgoingEdges(
+                  getNeighbourEntityType(environment.getArguments()).orElse(null),
+                  getNeighbourEntityScope(environment.getArguments()).orElse(null),
+                  Collections.emptyList()));
```

Comment:
So we're discarding the filters here - it'll work right now with the restriction to a single filterable type, but please leave a comment in all the places where this assumption is required